src/examples/mfem_common_plane.cpp

```diff
@@ -88,6 +88,8 @@ int main( int argc, char** argv )
   double lambda = 100000.0;
   // Lame parameter mu (shear modulus)
   double mu = 100000.0;
+  // kinematic penalty parameter (only for constant penalty)
```

Comment:
Since this is an example, a host-code may want to replicate this calculation as a 'guide' for what a constant penalty would be. Can you add in the comment what this calculation is?

Reply:
Is this calculation bulk modulus in terms of lambda and mu. Please note.